src/components/ProductCard.tsx

```diff
@@ -16,82 +17,57 @@ export default function ProductCard({ content }: { content: Content }) {
   const [selectedColour, setSelectedColour] = useState("black");
   const [cardImage, setCardImage] = useState(content.image[0]);
   const [isCardFlipped, setIsCardFlipped] = useState(true);
+  const [opacity, setOpacity] = useState(1);
 
-  useEffect(() => {
-    setCardImage(content.image[0]);
-  }, [content]);
+  const changeImage = (newImage: string) => {
+    setOpacity(0);
 
-  const toggleIsFlippedCard = () => {
+    setTimeout(() => {
+      setCardImage(newImage);
+      setOpacity(1);
+    }, 350);
+  };
+
+  const changeIsCardFlipped = () => {
     setIsCardFlipped(!isCardFlipped);
   };
 
   return (
-    <div className="flex flex-row flex-wrap justify-center">
-      <div className="bg-gray-100 group flex flex-col relative justify-center border-4 border-white">
-        <div className="z-30">
-          <Transition
-            show={!isCardFlipped}
-            className=""
-            enter="transform transition ease-in-out duration-300"
-            enterFrom="opacity-0 -translate-x-0"
-            enterTo="opacity-100 translate-x-0"
-            leave="transform transition ease-in-out duration-300"
-            leaveFrom="opacity-0 translate-x-0"
-            leaveTo="opacity-100 translate-x-0"
-          >
-            <button
-              className="group-hover:visible flex flex-row absolute top-0 right-0 py-1 px-2 text-xs bg-gray-300 z-20 "
-              onClick={toggleIsFlippedCard}
-            >
-              SHOW MORE
-              <Image
-                className="flex relative left-1"
-                src="/icons8-plus-50.png"
-                alt="X"
-                width={14}
-                height={14}
-                style={{ height: "15px", width: "15px" }}
-              />
-            </button>
-          </Transition>
-          <Transition
-            show={isCardFlipped}
-            className=""
-            enter="transform transition ease-in-out duration-300"
-            enterFrom="opacity-0 -translate-x-0"
-            enterTo="opacity-100 translate-x-0"
-            leave="transform transition ease-in-out duration-300"
-            leaveFrom="opacity-0 translate-x-0"
-            leaveTo="opacity-100 translate-x-0"
+    <div className=" group flex flex-col flex-wrap justify-center border-4 bg-gray-100 border-white">
+      <div className="flex flex-row relative justify-center">
+        <ProductCardShowMoreButton
+          isCardFlipped={isCardFlipped}
+          changeIsCardFlipped={changeIsCardFlipped}
+          changeImage={changeImage}
+          content={content}
+        />
+      </div>
+      <div className="flex flex-col w-[413.33px] h-[508px] items-center relative bg-gray-100">
+        <div className="top-10 relative">
+          <div
+            className={`${
+              isCardFlipped
+                ? "[transform:-rotateY(180deg)]"
```

Comment:
I would extract this and have the ternary dictate just the direction then constructing the class after that